relay_handler.js

```diff
@@ -328,6 +335,12 @@ function forwardTo(conn) {
     self.handleFrameLazily(self.reqFrame);
     self.reqFrame = null;
 
+    for (var i = 0; i < self.reqContFrames.length; i++) {
+        self.handleFrameLazily(self.reqContFrames[i]);
+    }
```

Comment:
I'll leave it here; it's very order specific around

 - `self.reqFrame = null;
 - `for loop`
 - `array.length = 0;`

A little fragile..